docs: per-agent reference with icons, commands, and labels (#1017)
konflux-ci/fullsend: docs/agents/prioritize.md

|---------|-------|--------|
| `/fs-prioritize` | Issue comment | Runs RICE scoring on the issue |

The `/fs-prioritize` command does not accept arguments. It scores the issue
using the current content, comments, and any available `customer-research`
skill data.

## Control labels

The prioritize agent does not apply or consume control labels. It reads the
issue content and produces a structured score — the post-script updates the
project board directly.

## Configuration and extension

Detailed harness-level customization is coming soon. Today, the best way to
influence how the prioritize agent behaves on your repository is by adding
instructions and skills to the repo itself. See
[Customizing Agents with Skills](../guides/user/customizing-with-skills.md).

### Example: Providing a customer-research skill

The prioritize agent looks for a `customer-research` skill and, when available,
uses it to inform Reach and Impact scores. This is not a command — it's an
agent skill that the prioritize agent invokes automatically during scoring.

To provide one, create a skill directory in your target repository at
`.claude/skills/customer-research/` with a `SKILL.md` and any helper scripts

> the agent can run to dynamically gather data:

Author: We should recommend a handy skill creator skill like anthropics own or the superpowers one, although most people already know those.

Author: Fixed in 7c740fd

```
your-repo/
  .claude/skills/customer-research/
    SKILL.md
    query-salesforce.sh
    search-drive.sh
```

> **`.claude/skills/customer-research/SKILL.md`:**

Author: Here, and everywhere, we should recommend storing these in `.agents/` and then symlinking to that from `.claude/` for portability.

Author: Fixed in 7c740fd

```markdown
---
name: customer-research
description: >-
  Gather customer context from Salesforce and Google Drive to inform
  RICE prioritization scoring.
---

# Customer Research

Research customer context for the issue being prioritized.

## Step 1: Check Salesforce for affected accounts

Run the Salesforce query script to find accounts with support cases
related to this issue's feature area:

>     bash .claude/skills/customer-research/query-salesforce.sh "SEARCH_TERM"

Author: Hm, we should recommend moving the scripts into `scripts/` in the skills dir (i.e. `.claude/skills/customer-research/scripts/query-salesforce.sh`) and then exhorting the agent to use it just with `scripts/query-salesforce.sh` and adding `Bash(scripts/query-salesforce.sh...)` to the allowed-tools of the skill (for local use outside of fullsend).

Author: Fixed in 7c740fd

# Prioritize Agent

<img src="icons/prioritize.png" alt="Prioritize agent icon" width="80">

Scores a GitHub issue using the RICE framework (Reach, Impact, Confidence, Effort) and produces structured scores with reasoning for project board ranking.

## How the agent works

The prioritize agent is triggered after triage. It fetches the issue and all its context, then evaluates it across the four RICE dimensions. It can invoke customer-research skills to gather additional signal about reach and impact. The output is a structured JSON result with per-dimension scores and written reasoning, which the post-script uses to update the project board.

> The agent runs in a read-only sandbox. It cannot modify issues or push code — it only produces a scored result.

Reviewer: This phrase is weird here. It reinforces the previous paragraph, but I don't think it adds anything new. Also it is incoherent: how it is ready-only but then sets a score and a comment on the issue? I get the point, it just reads issues and does not modify them because it can't because the sandbox prevents it (which I'm not completely sure, as traffic to GH is granted). I would remove it.

Also, isn't this supposed to be a list of actions?

Author: Hm, ok. +1 to remove it.

Author: Removed in 0ac7886.

## How it helps

- Issues are ranked consistently using the same framework, reducing bias from whoever happens to see them first.
- Scoring reasoning is transparent and auditable — anyone can read why an issue was ranked the way it was.
- Project boards stay sorted by value, so humans can focus on the highest-impact work first.

> ## Commands

Reviewer: You mention in the previous section that is triggered after triage, does that mean that it reacts to some label? Sounded automatic instead of having to type `/fs-prioritize`.

Author: Yeah, is this actually true though? We should verify that the prioritize agent actually triggers after triage. I think it may only be cron-based (although you can explicitly invoke it with `/fs-prioritize`; to be confirmed).

Author: Fixed in e00de60. It's not triggered after triage — it runs on a schedule (polling for unscored/stale issues) or on-demand via /fs-prioritize.

| Command | Where | Effect |
|---------|-------|--------|
| `/fs-prioritize` | Issue comment | Runs RICE scoring on the issue |

The `/fs-prioritize` command does not accept arguments. It scores the issue
using the current content, comments, and any available `customer-research`
skill data.

## Control labels

The prioritize agent does not apply or consume control labels. It reads the
issue content and produces a structured score — the post-script updates the
project board directly.

## Configuration and extension

> ### Example: Providing a customer-research skill

Reviewer: I don't think this is an "example", I would call it

```suggestion
### Skill: `customer-research`
```

Author: Yeah, good call

Author: Applied in 0ac7886.

The prioritize agent looks for a `customer-research` skill and, when available,
uses it to inform Reach and Impact scores. This is not a command — it's an
agent skill that the prioritize agent invokes automatically during scoring.

> 
> To provide one, create a skill directory in your target repository at
> `.agents/skills/customer-research/` with a `SKILL.md` and any helper scripts
> organized in a `scripts/` subdirectory. Then symlink `.claude/skills` to
> `.agents/skills` so the skill is discoverable by both fullsend and any local
> agent tooling:

Reviewer: ```suggestion
The prioritize agent looks for a `customer-research` skill and, when available,
uses it to inform Reach and Impact scores. To provide it, create a skill directory
in your target repository at `.agents/skills/customer-research/` with a `SKILL.md` and
any helper scripts
organized in a `scripts/` subdirectory. Then symlink `.claude/skills` to
`.agents/skills` so the skill is discoverable by both Fullsend and any local
agent tooling:
```

Author: +1, I like yours better

Author: Applied in 0ac7886.

```
your-repo/
  .agents/skills/customer-research/
    SKILL.md
    scripts/
      query-salesforce.sh
      search-drive.sh
  .claude/skills -> ../.agents/skills
```

> **Tip:** You can scaffold this structure using a skill creator skill like
> Anthropic's built-in `/skill` or the superpowers `writing-skills` skill.

> **`.agents/skills/customer-research/SKILL.md`:**

Reviewer: ```suggestion
**Example `.agents/skills/customer-research/SKILL.md`:**
```

Author: Applied in 0ac7886.

> ```markdown

Reviewer: This file is redenred a bit weird: 

<img width="455" height="630" alt="Image" src="https://github.com/user-attachments/assets/15bc549d-8302-4c80-ba99-bc32e8787102" />

Author: Yeah, drop the salesforce example and let's revisit the gdrive example.

Author: Dropped the script examples in 0ac7886.

```

**`.agents/skills/customer-research/scripts/query-salesforce.sh`:**

```bash
#!/usr/bin/env bash
set -euo pipefail
TERM="$1"
curl -sf -H "Authorization: Bearer $SFDC_TOKEN" \
  "$SFDC_URL/services/data/v59.0/query/" \
  --data-urlencode "q=SELECT Account.Name, Account.Tier__c, Count(Id) cnt
    FROM Case WHERE Subject LIKE '%${TERM}%'
    GROUP BY Account.Name, Account.Tier__c
    ORDER BY cnt DESC LIMIT 20"
```

**`.agents/skills/customer-research/scripts/search-drive.sh`:**

```bash
#!/usr/bin/env bash
set -euo pipefail
TERM="$1"
curl -sf -H "Authorization: Bearer $GDRIVE_TOKEN" \
  "https://www.googleapis.com/drive/v3/files" \
  --data-urlencode "q=fullText contains '${TERM}' and mimeType='application/vnd.google-apps.document'" \
  -d "fields=files(name,modifiedTime,webViewLink)" \
  -d "orderBy=modifiedTime desc" \
  -d "pageSize=5"
```

This gives the prioritize agent concrete data to distinguish between "one user
wants this" (Reach 0.25) and "three strategic accounts have filed support cases
about it" (Reach 2.0), instead of guessing from the issue text alone. The
scripts run dynamically during each prioritization, so scores reflect current

> customer data.

Reviewer: I would move this section to the top and make it talk about the intent of the skill.

Author: Applied in 0ac7886.
